spec.md

```diff
@@ -109,8 +109,19 @@ This header allows brokers to reject requests from marketplaces for versions the
 
 ## Authentication
 
-The marketplace MUST authenticate with the service broker using HTTP
-basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
+While the communication between a platform and broker
+MAY be unsecure, it is RECOMMENDED that all communications between a
+platform and a broker are secured via TLS and authenticated.
+
+Unless there is some out of band communication and agreement between a
+platform and a service broker, the marketplace MUST authenticate with the
+service broker using HTTP basic authentication (the `Authorization:` header)
+on every request. This specification does not specify how platform and brokers
+agree on other methods of authentication.
+
+Regardless of the authentication mechanism that is used, the broker MUST
+validate the credentials from the platform and MUST return a
```

Comment:
"validate the credentials" - are certificates/tokens considered credentials? 

Perhaps "authenticate the request using the predetermined authentication mechanism" is clearer?